### PR TITLE
Fuck Greytiders: Buffs the Chef's CQC

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
chef can cqc in all of service now
## Why It's Good For The Game

fuck greytiders, quit killing the robots

## Changelog
:cl:
balance: Chef CQC now works in all of Service, to curb greytiding the robots.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
CHEF CQC'S THE CURATOR IN THEIR OWN LIBRARY. ABSOLUTELY WINNING

**Comment by CyoTheProtogen** [comment]
speedmerge

**Comment by Kylerace** [comment]
would it make more sense to only expand cqc to bar?

**Comment by Serxule** [comment]
hell yeah i can finally cqc everyone, also does this count the bar/theater too?

**Comment by maxymax13** [comment]
strangling the botanist until he makes the vegetables i asked him for instead of 5 tonnes of weed

**Comment by SgtHunk** [comment]
A JANITOR IN HIS CUSTODIAL CLOSET? SOUNDS LIKE YOU'RE JUST A FUCKING GREYTIDER

**Comment by maxymax13** [comment]
imagine being a chap just chilling in your chapel and then a chef with red eyes busts the door down and kicks your ass

**Comment by Fikou** [comment]
just expand it to bar, this is dumb

**Comment by ardentarclight** [comment]
phenomenally based

**Comment by Qustinnus** [comment]
good pr but make it bar and kitchen only

**Comment by itseasytosee** [comment]
Power Crêpe

**Comment by oklews** [comment]
Literally fucking murdering psychologists in medbay because they are service, or lawyers because they are service.

**Comment by Serxule** [comment]
> Literally fucking murdering psychologists in medbay because they are service, or lawyers because they are service.

I don't think their rooms count as service though

File: code/datums/martial/cqc.dm [hunk 1]
```diff
@@ -225,7 +225,7 @@
 ///Subtype of CQC. Only used for the chef.
 /datum/martial_art/cqc/under_siege
 	name = "Close Quarters Cooking"
-	var/list/valid_areas = list(/area/service/kitchen)
+	var/list/valid_areas = list(/area/service)
```
Comment (coiax) [inline code/datums/martial/cqc.dm:228]: Just expand it to kitchen and bar. The chef shouldn't know Close Quarters Cooking when in the Janitor's little hideaway.

**Review by coiax — CHANGES_REQUESTED** [review]

File: code/datums/martial/cqc.dm (hunk 1 above)
Comment (brotherbeyondd) [reply code/datums/martial/cqc.dm:228]: What about botany? They're almost always next to the kitchen.

Comment (Fikou) [reply code/datums/martial/cqc.dm:228]: > What about botany? They're almost always next to the kitchen.

t. chef main

**Comment by Iamgoofball** [comment]
@oklews Psychologist areas should be pathed under Medical. It's not my problem.

**Comment by ATH1909** [comment]
thanks for reducing this to just kitchen/bar-relevant areas instead of all service areas, this gets a 👍 from me now (not that anyone cares...)

**Comment by Fikou** [comment]
cafeteria has never been used/is not currently used for any area related to the chef, i think its only used on metastation for the jim hortons coffee shop which is right in the middle of the primary hallway, please do not give chefs cqc in the middle of the primary hallway

**Pull request merged**